### PR TITLE
Add call for speakers page

### PR DESCRIPTION
**Pull request by Kriesse**
- [x] add page
- [x] update copy and links for 2019
- [x] add paragraph on closing date etc
- [x] add link to form
- [x] add CfS to main navigation

### CODE REVIEW

**Comment by Lucie-H** [comment]
@Kriesse I don't see the paragraph about closing date and the link to the form? 😅🙈

**Comment by Kriesse** [comment]
@Lucie-H sorry my fault, try again?

File: contents/copy/cfs/intro.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Call For Speakers
+introText: >
+  CSSconf EU is targeted at CSS professionals, CSS creatives, CSS aficionados. It doesn’t matter whether you are a professional front end developer, a designer who practices CSS magic in their spare time, or you just love CSS and want to share your knowledge. We’re interested in topics that make us go, "Oh, wow! Yes! Cool! I want to hear about that!"
```
Comment (sfrdmn) [inline contents/copy/cfs/intro.md]: "CSS professionals, creatives, and aficionados"

Comment (sfrdmn) [inline contents/copy/cfs/intro.md]: "Oh, wow! Yes! Cool!" 😆👍

File: contents/copy/cfs/selection-process.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Selection Process
+
+1. All submissions are anonymized, so that there is no bias towards the submitter.
```
Comment (sfrdmn) [inline contents/copy/cfs/selection-process.md]: No comma, also maybe "to ensure there is no"

File: contents/copy/cfs/we-can-help.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## We can help
+
+We understand that not everyone is a natural-born talent on stage, or can produce kick-ass slide decks, or is a live-demo master. Not everyone knows they have something great to talk about.
```
Comment (sfrdmn) [inline contents/copy/cfs/we-can-help.md]: kick-ass slide decks? 🤔

Comment (Kriesse) [reply contents/copy/cfs/we-can-help.md]: please edit and make less awkward 🙏

File: contents/copy/cfs/selection-process.md (hunk 2 above)
Comment (Kriesse) [reply contents/copy/cfs/selection-process.md]: please edit?

File: contents/copy/cfs/intro.md (hunk 1 above)
Comment (Kriesse) [reply contents/copy/cfs/intro.md]: 🙃

Comment (Kriesse) [reply contents/copy/cfs/intro.md]: can you edit and fix?

**Pull request merged**